packages/contracts-bedrock/contracts/libraries/BytesUtils.sol

```diff
@@ -3,19 +3,27 @@ pragma solidity ^0.8.9;
 
 /**
  * @title BytesUtils
+ * @notice BytesUtils is a library for manipulating byte arrays.
  */
 library BytesUtils {
-    /**********************
-     * Internal Functions *
-     **********************/
-
+    /**
+     * @custom:attribution https://github.com/GNSPS/solidity-bytes-utils/blob/6458fb2780a3092bc756e737f246be1de6d3d362/contracts/BytesLib.sol#L228-L295
```

Comment:
I think it would be nice to include `@custom:attribution` tags for any code that we're using via external sources. Then we can easily link to attribution in the generated docs.